### PR TITLE
chore: Localization of Superset pt. 3

### PR DESCRIPTION
**Pull request by artemonsh**
### SUMMARY

This PR is the continuation of the previous localization PR https://github.com/apache/superset/pull/22772. More charts and elements are localized. However, there is still some work to do in the future.

Difficult cases are marked with `TODO: localize` comment.

### ADDITIONAL INFORMATION

- [ ] Has associated issue:
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: superset-frontend/src/explore/components/ControlHeader.tsx [hunk 1]
```diff
@@ -105,7 +105,7 @@ const ControlHeader: FC<ControlHeaderProps> = ({
         {description && (
           <span>
             <Tooltip
-              id={`${t('description')}-tooltip`}
```
Comment (rusackas) [inline superset-frontend/src/explore/components/ControlHeader.tsx:108]: good catch

File: superset-frontend/src/views/CRUD/data/dataset/DatasetList.tsx [hunk 2]
```diff
@@ -343,7 +343,7 @@ const DatasetList: FunctionComponent<DatasetListProps> = ({
           row: {
             original: { kind },
           },
-        }: any) => kind[0]?.toUpperCase() + kind.slice(1),
+        }: any) => (kind === 'physical' ? t('Physical') : t('Virtual')),
```
Comment (rusackas) [inline superset-frontend/src/views/CRUD/data/dataset/DatasetList.tsx:346]: I suppose this is less flexible, e.g. we'd be in trouble if we ever added another `kind` but I think it's fine for now.

File: superset-frontend/src/SqlLab/components/QueryTable/index.tsx [hunk 3]
```diff
@@ -76,17 +76,22 @@ const QueryTable = ({
...
+    results: t('Results'),
+    actions: t('Actions'),
   };
+
   const columnsOfTable = useMemo(
     () =>
       columns.map(column => ({
         accessor: column,
-        Header: () => setHeaders(column),
+        Header: QUERY_HISTORY_TABLE_HEADERS_LOCALIZED[column],
```
Comment (rusackas) [inline superset-frontend/src/SqlLab/components/QueryTable/index.tsx:94]: This is the only part that makes me a little nervous. I haven't done any real spelunking in the code here, but are we confident that these are the only header strings we'll need to worry about for the foreseeable future?

I wonder if it's safer to leave the function in place as a backup, i.e.

```suggestion
        Header: QUERY_HISTORY_TABLE_HEADERS_LOCALIZED[column] || setHeaders(column),
```

**Review by rusackas — APPROVED** [review]
Left a couple comments, but approving and pinging @betodealmeida @AAfghahi @lyndsiWilliams in case they want to traipse through the code/comments as well (particularly the bit about the headers... I'm not sure how permanent those column headers are). 

Thanks for all the helpful improvements here! I love to see this all DRY-ing up!

**Comment by rusackas** [comment]
Oh, and it seems we have some conflicts (ugh) so a rebase is in order. Thanks again, we'll try to stay on top of this so you don't have to rebase often :D

**Review by michael-s-molina — APPROVED** [review]

**Pull request merged**